src/main/scala/com/fulcrumgenomics/personal/nhomer/HasSequence.scala

```diff
@@ -0,0 +1,157 @@
+/**
+  * Copyright (c) 2016, Fulcrum Genomics LLC
+  * All rights reserved.
+  *
+  * Redistribution and use in source and binary forms, with or without
+  * modification, are permitted provided that the following conditions are met:
+  *
+  * 1. Redistributions of source code must retain the above copyright notice,
+  * this list of conditions and the following disclaimer.
+  *
+  * 2. Redistributions in binary form must reproduce the above copyright notice,
+  * this list of conditions and the following disclaimer in the documentation
+  * and/or other materials provided with the distribution.
+  *
+  * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+  * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+  * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+  * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+  * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+  * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+  * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+  * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+  * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  * POSSIBILITY OF SUCH DAMAGE.
+  */
+
+package com.fulcrumgenomics.personal.nhomer
+
+import java.nio.file.Path
+
+import com.fulcrumgenomics.cmdline.{JeanLucTool, Personal}
+import dagr.commons.CommonsDef.PathToBam
+import dagr.commons.io.Io
+import dagr.sopt._
+import htsjdk.samtools._
+import htsjdk.samtools.metrics.{MetricBase, MetricsFile}
+import htsjdk.samtools.util._
+import picard.PicardException
+
+import scala.io.Source
+
+@clp(
+  description = "Searches for DNA sequences in the read pairs.",
+  group = classOf[Personal]
+)
+class HasSequence
+( @arg(doc = "Input SAM or BAM.") val input: PathToBam,
```

Comment:
1. nope, I do like writing `input` and `output` for a single tool.  
2. `-i` and `-o` will just work in this instance since there are no other args that start with either `i` or `o`